splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/pages/Input/InputPage.jsx

```diff
@@ -26,420 +25,574 @@ function InputPage({ isInput, serviceName }) {
+                    modifiedResponse.push({
+                        ...val.content,
+                        id: val.id,
+                        name: val.name
+                    });
                 });
-            });
-            setRowData(modifiedResponse);
-            setLoading(false);
-        }
-    };
-
-    /**
-     *
-     * @param row {Object} row
-     */
-    const changeStatus = (row) => {
-        let oldData = rowData;
-        const index = oldData.findIndex((val) => {
-            return val.id == row.id;
+                obj[service.name] = modifiedResponse;
+            }
         });
-        if (index != -1) {
-            oldData[index].disabled = !oldData[index].disabled;
-        }
-        setRowData((data) => [...oldData]);
-    };
-
-    const handleRequestClose = () => {
-        setOpen(false);
-    }
-
-    // temprary button for test Form
-    const generateModalDialog = () => {
-        if (open) {
-            return (
-                < EntityModal
-                    isInput
-                    open={open}
-                    handleRequestClose={handleRequestClose}
-                    handleSavedata={null}
-                    serviceName="example_input_one"
-                    mode="CREATE"
-                    currentInput={null}
-                    formLabel="Add Input"
-                />
-            );
-        }
-            return null;
-    }
-
-    
-
-    const handleRequestOpen = () => {
-        setOpen(true);
+        setRowData(obj);
+        setLoading(false);
     }
 
     return (
         <>
-            {loading ? (
-                <WaitSpinnerWrapper size="large" />
-            ) : (
-                <>
-                    <TitleComponent>{title}</TitleComponent>
-                    <SubTitleComponent>{description}</SubTitleComponent>
-                    <Button onClick={handleRequestOpen} appearance="primary" label="Add Input" />
-                    <hr />
-                    <TableCaptionComponent>
-                        <div className="table-caption-inner">
-                            {rowData.length} Input {rowData.legnth > 1 && <span>s</span>}
-                        </div>
-                    </TableCaptionComponent>
-                    <Table
-                        isInput={isInput}
-                        serviceName={serviceName}
-                        data={rowData}
-                        handleToggleActionClick={(row) => changeStatus(row)}
-                    />
-                    
-                    {generateModalDialog()} 
-                </>
-            )}
+            <TitleComponent>{title}</TitleComponent>
+            <SubTitleComponent>{description}</SubTitleComponent>
+            { loading ?
+                <WaitSpinnerWrapper size="large" /> :
+                <TableWrapper
+                    isInput={isInput}
+                    isInput={isInput}
+                    isInput={isInput}
```

Comment:
@tbalar-splunk Please remove duplicate props.

Reply:
It's removed